aspnetcore/fundamentals/minimal-apis/security.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Configuring authentication for minimal APIs
+author: captainsafia
+description: Learn how to configure authentication and authorization in minimal API apps
+ms.author: safia
+monikerRange: '>= aspnetcore-7.0'
+ms.date: 10/17/2022
+uid: fundamentals/minimal-apis/security
+---
+
+# Authentication and authorization in minimal APIs
+
```

Comment:
nit: Having nothing between the title and the first header doesn't look very appealing on the docs site.